### ISSUE TITLE
Things that bothered me

### ISSUE THREAD

**Issue by 22Emily** [wonderunit/storyboarder#2474]
I've been using the program for about a week now and I really like it but I'm frustrated that the Export pdf or Export video and Export to GIF in the app has a watermark. 

Or if it's fair All  **Options 3**
- [ ] options 1 | **no** watermark (**absolutely necessary**)
- [ ] options 2 | have watermark
- [ ] options 3 | Custom watermark.

(Can you **remove** it? | Having a **watermark** makes my work more difficult.because I want to finish it in the program)

Given in Frame by Frame is difficult for video framing. (want to show the **time in detail**)
- [ ] options 1 | mm.ss / minute, second (Default)
- [ ] options 2 | hh.mm.ss.**ms** / Hour, minute, second, **millisecond**
- [ ] options 3 | hh.mm.ss.**ff** / Hour, minute, second, **Frames**

In video, Final Cut Pro and premiere Pro are **amazing.and no watermark**

**Add Custom Video**
- [ ] Format : Ex.mp4 mov mkv
- [ ] Video Codec : Ex. H.264 H.265
- [ ] Resolution : Ex. 100%-1000% size and Up size 1080p 2K 4K but Remain Aspect Ratio
- [ ] Show Time Render Video

**Key Commands**
- [ ] Add Custom Shortcut Keyboard
- [ ] Add Custom Shortcut Keyboard add menu bar(**Mac OS**)

Key Commands UI very beautiful

**Theme**
- [ ] Add Custom Theme Ex.Black | Gray | White

Can finish in the program

I use **ver.`3.0.0`**

----------------------------------------------------------------------------------------------------------------------------
**Things to improve**
- [ ] Subject 1 : Watermark
- [ ] Subject 2 : Time & Frame-by-frame is difficult for framing video.
- [ ] Subject 3 : Add Custom Video
- [ ] Subject 4 : Custom Shortcut Keyboard
- [ ] Subject 5 : Custom Theme
----------------------------------------------------------------------------------------------------------------------------
                                                Can you fix it?

**Comment by lazyradly** [comment]
Agreed, for a free and open source software it should not have a watermark or at least give you the option to disable the watermark for video exports. Very unusual that they decided to add it in without the option to remove it.
